tests/validate-hostname.yaml

```diff
@@ -46,6 +46,10 @@ tests:
   - host: "xn--bcher-kva.ch"
     valid: true
 
+  # host ending with a .
+  - host: "services."
+    valid: true
+
```

Comment:
I've never seen an irc hostname or servername end with `.`, so including this here as a 'valid' result feels really dodgy. all my software would reject this as invalid, even though the RFC specifies that this is a totally legitimate hostname.

Reply:
Example on freenode:

```
* [ChanServ] (ChanServ@services.): Channel Services
* [ChanServ] services. :Atheme IRC Services
* [ChanServ] is a Network Service
* [chanserv] End of WHOIS list.
```